pkg/drivers/qemu/qemu.go

```diff
@@ -467,7 +467,7 @@ func (d *Driver) Start() error {
 	if stdout, stderr, err := startFunc(startProgram, startCmd...); err != nil {
 		fmt.Printf("OUTPUT: %s\n", stdout)
 		fmt.Printf("ERROR: %s\n", stderr)
-		return err
+		exit.Message(reason.QemuSocketVmnetPermission, stderr+"\n\n")
```

Comment:
The error needs to be checked before we know that it's a permission error, the error could be something else unrelated

Reply:
Thanks for your feedback, @spowelljr . I added a check for the error message.